elements/pl-checkbox/pl-checkbox.py

```diff
@@ -481,8 +494,9 @@ def test(element_html, data):
                 raise ValueError(f'Unknown value for partial_credit_method: {partial_credit_method}')
         else:
             score = 0
+        submitted_keys_to_feedback = {option['key']: option['feedback'] for option in data['params'][name] if option['key'] in ans}
         data['raw_submitted_answers'][name] = ans
-        data['partial_scores'][name] = {'score': score, 'weight': weight}
+        data['partial_scores'][name] = {'score': score, 'weight': weight, 'submitted_keys_to_feedback': submitted_keys_to_feedback}
```

Comment:
I'd rather we use just `feedback` here . That would be consistent with how elements work, including those with multiple pieces of feedback, for instance `pl-matrix-component-input`:

https://github.com/PrairieLearn/PrairieLearn/blob/5b997eabb69a0b5707f6b0e2ab2d615069455756/elements/pl-matrix-component-input/pl-matrix-component-input.py#L381-L413